go/oasis-test-runner/scenario/e2e/runtime/multiple_runtimes.go

```diff
@@ -104,11 +104,26 @@ func (sc *multipleRuntimesImpl) Fixture() (*oasis.NetworkFixture, error) {
 				MinWriteReplication:     1,
 				MaxApplyWriteLogEntries: 100_000,
 				MaxApplyOps:             2,
-				MinPoolSize:             1,
 			},
 			AdmissionPolicy: registry.RuntimeAdmissionPolicy{
 				AnyNode: &registry.AnyNodeRuntimeAdmissionPolicy{},
 			},
+			Constraints: map[scheduler.CommitteeKind]map[scheduler.Role]registry.SchedulingConstraints{
+				scheduler.KindComputeExecutor: {
+					scheduler.RoleWorker: {
+						MinPoolSize: &registry.MinPoolSizeConstraint{
+							Limit: uint16(executorGroupSize),
+						},
+					},
+				},
+				scheduler.KindStorage: {
+					scheduler.RoleWorker: {
+						MinPoolSize: &registry.MinPoolSizeConstraint{
+							Limit: 1,
+						},
```

Comment:
a similar MaxNodes is not added here

Reply:
There was no MaxNodes before. But there was a MinPoolSize.